cli/pcluster/easyconfig.py

```diff
@@ -290,3 +303,14 @@ def _is_config_valid(args, config):
         if is_file_ok:
             return True
         return False
+
+
+def _is_aws_batch(scheduler):
+    """
+    Return true if the scheduler is awsbatch
+    :param scheduler: the scheduler to check
+    :return: true if the scheduler is awsbatch
+    """
+    if scheduler == "awsbatch":
+        return True
```

Comment:
you can write:
`return scheduler == "awsbatch"`
so I think we can avoid this simple function that just perform an equality test